lib/medtronic/processData.js

```diff
@@ -902,17 +903,19 @@ var buildTempBasalRecords = function (records) {
         .set('index', record.index)
         .set('jsDate',record.jsDate);
 
-      if(percentage) {
+      if(percentage != null) {
```

Comment:
In a lot of the driver code we use `!= null` as a check for both `null` and `undefined`.